### PR TITLE
Updated slack guide

### PR DESCRIPTION
**Pull request by ReedCopsey**

### CODE REVIEW

File: guides/slack/index.md [hunk 1]
```diff
@@ -4,22 +4,45 @@ title: Guide - F# on Slack | The F# Software Foundation
 headline: Guide - F# Slack Teams
 ---
 
-[Slack](https://slack.com/) is a persistent chat service. Some of its features are:
+## F# Software Foundation Slack
+
+[Slack](https://slack.com/) is a persistent chat service. 
+
+Members of the F# Software Foundation can be invited to the [F# Software Foundation](https://fsharp.slack.com/) team. To join, become a member of the F# Software Foundation by [visiting this page](http://foundation.fsharp.org/join). Once you are a member, you will automatically be queued to receive an invite to the Foundation's Slack team.
```
Comment (dsyme) [inline guides/slack/index.md]: You might add something about how to join the team if you didn't get/notice the first invitation

File: guides/slack/index.md [hunk 2]
```diff
@@ -4,22 +4,45 @@ title: Guide - F# on Slack | The F# Software Foundation
...
+Members of the F# Software Foundation can be invited to the [F# Software Foundation](https://fsharp.slack.com/) team. To join, become a member of the F# Software Foundation by [visiting this page](http://foundation.fsharp.org/join). Once you are a member, you will automatically be queued to receive an invite to the Foundation's Slack team.  
+
+Once you sign into the Slack team, there are various channels containing topics in which you can participate:
+
+* [#general](https://fsharp.slack.com/messages/general/): General discussions on any topic, including social conversations for members of the Foundation
+* [#beginners](https://fsharp.slack.com/messages/beginners/): Questions and answers for beginners coming to F# and the F# Community
+* [#code](https://fsharp.slack.com/messages/code/): Code-focused discussion on any F# related topic
+* [#help](https://fsharp.slack.com/messages/help/): Help on any F# related topic
+* [#web](https://fsharp.slack.com/messages/web/): Discussion focused on using F# with web based technologies
+* [#compiler](https://fsharp.slack.com/messages/compiler/): Discussion surrounding the F# compiler and FSharp.Core
```
Comment (dsyme) [inline guides/slack/index.md]: FYI I don't (yet) monitor this slack channel.  I suppose I should, but it's yet another place to monitor and/or get notifications from....

File: guides/slack/index.md [hunk 3]
```diff
@@ -4,22 +4,45 @@ title: Guide - F# on Slack | The F# Software Foundation
...
+* [#editor_support](https://fsharp.slack.com/messages/editor_support/): Discussion focused on adding IDE support for F#
+* [#jobs](https://fsharp.slack.com/messages/jobs/): Job and career related discussions for F# Software Foundation members
+* [#dotnetcore](https://fsharp.slack.com/messages/dotnetcore/): Discussion around F# support for .NET Core, as well as porting libraries
+* [#speakers](https://fsharp.slack.com/messages/speakers/): Discussion for people who speak about F#, as well as the [F# Software Foundation Speakers' Program](http://community.fsharp.org/speakers_program)
+* [#mentorship](https://fsharp.slack.com/messages/mentorship/): Discussion about the [F# Software Foundation Mentorship Program](http://fsharp.org/mentorship/index.html)
+* [#user_group_leaders](https://fsharp.slack.com/messages/user_group_leaders/): Discussion for leaders of user groups
+
+
+If you don't receive the invite within 48 hours, or if you have any questions, please send an email to info@fsharp.org to follow up.
```
Comment (ReedCopsey) [inline guides/slack/index.md]: @dsyme mentioned here.  shouldn't be an issue with new system though.

File: guides/slack/index.md (hunk 1 above)
Comment (ReedCopsey) [inline guides/slack/index.md]: @dsyme yes, though it's a good option for people doing discussion in real time... has helped with some PRs already ;)  I think it's more for people to have a venue to get immediate help and discussion when they're working on something.

**Pull request merged**